pyfranca/franca_parser.py

```diff
@@ -583,6 +585,67 @@ def p_map_def(p):
 
     # noinspection PyIncorrectDocstring
     @staticmethod
+    def p_constant_def(p):
+        """
+        constant_def : CONST type ID '=' INTEGER
+        """
+        if p[2].name not in [  "Int8",  "Int16",  "Int32",  "Int64",
```

Comment:
Your are rigtht I also think about that. But if I only store type and value for a constant, the processor needs to determine the value type in order to check if it matches teh constant type. With type() I can detremine type for string, boolean, integer and float values but not double. double and float is the same in python. I want to check that a float constant is not initilized with an double value because of loosing precision. If I follow your suggestion I need to store the type of the value. Maybe I do not see the easy way and my approach is too difficult. I think about that.